Fixed potential errors if the state is undefined

BenjaminVettori opened the pull request:
This caused e.g. errors when the StateClear actions of ngxs-reset-plugin is used.
Fixes #16, where the state is completely removed.
By ignoring undefined and treating it as count 0, it will continue to work for such cases.

BenjaminVettori committed:
Fixed potential errors if the state is undefined

This caused e.g. errors when the StateClear actions of ngxs-reset-plugin is used.

joaqcid commented on src/lib/actions-executed.state.ts:
```diff
@@ -26,7 +26,8 @@ export class ActionsExecutedState implements NgxsOnInit, OnDestroy {
 
-                    let count = getState()[actionType] || 0;
+                    var state = getState();
+                    let count = state ? state[actionType] || 0 : 0;
```
hi, can you change it to optional chaining `let count = getState()?.[actionType] || 0;`

BenjaminVettori replied:
Thank you for the hint @joaqcid, i fixed it according to your suggestion

BenjaminVettori committed:
replaced ternary operator with optional chaining

BenjaminVettori committed:
Use or instead of null coalescing operator

?? treats only null and undefined, while || will also lead to 0, when the left value is an empty string

joaqcid commented:
@BenjaminVettori released on v0.2.0
